exercises/en/test_08_08.py

```diff
@@ -0,0 +1,14 @@
+def test():
```

Comment:
In this section can you change this line "Fit your grid search on the training data" in the exc and solution file so it's 

"Fit your model on the training data"